environment.yml

```diff
@@ -1,9 +1,8 @@
-name: eureka
+name: eureka_temp
 channels:
     - conda-forge
-    - defaults
 dependencies:
-    - python==3.13.0
+    - python==3.12
```

Comment:
Why are we rolling back from 3.13 to 3.12?  Also, the setup.cfg uses python >=3.11,<3.14.

Reply:
Python 3.13 doesn't allow numpy==1.26.4 which I think one of our package versions need or needed (see #809).